x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -70,6 +75,23 @@ export class HeadlessChromiumDriver {
       interceptedCount = interceptedCount + (isData ? 0 : 1);
     });
 
+    if (this.networkPolicy.enabled) {
```

Comment:
This is an escape hatch to bail out of this entirely if some edge-case exists that I'm not handling.